Controller/FormWebsiteController.php

```diff
@@ -260,17 +268,37 @@ protected function getAttributes($attributes, StructureInterface $structure = nu
         return $this->attributes;
     }
 
-    /**
-     * Get type class.
-     *
-     * @param string $key
-     *
-     * @return string
-     */
-    private function getTypeClass($key)
+    private function getTypeClass(string $key): string
+    {
+        return $this->staticForms[$key]['class'];
+    }
+
+    protected function getRequestStack(): ?RequestStack
+    {
+        return $this->get('request_stack');
+    }
+
+    protected function getFormRegistry(): ?FormRegistryInterface
```

Comment:
make them private. FormRegistry is not optional in our case so returnType is `: FormRegistryInterface`